python/pylibvw.cc

```diff
@@ -210,6 +220,145 @@ class OptionManager
   }
 };
 
+class PyCppCallback
+{
+private:
+  void* base_learner;
```

Comment:
Does this need to be a `void*`? Can you store the member as a `base_learner*`?

Reply:
done